Dockerfile

```diff
@@ -1,32 +1,8 @@
-FROM debian:stretch-slim
-
-ENV BITCOIN_VERSION 0.20.2
-ENV BITCOIN_URL https://download.bitcoinabc.org/$BITCOIN_VERSION/linux/bitcoin-abc-$BITCOIN_VERSION-x86_64-linux-gnu.tar.gz
-
-ARG USER_ID
-ARG GROUP_ID
-
-ENV HOME /home/bitcoin
-
-# add user with specified (or default) user/group ids
-ENV USER_ID ${USER_ID:-1000}
-ENV GROUP_ID ${GROUP_ID:-1000}
-
-RUN groupadd -g ${GROUP_ID} bitcoin \
-	&& useradd -u ${USER_ID} -g bitcoin -s /bin/bash -m -d ${HOME} bitcoin
-
-RUN set -ex \
-	&& apt-get update \
-	&& apt-get install -qq --no-install-recommends ca-certificates wget gosu \
-	&& rm -rf /var/lib/apt/lists/*
-
-# install bitcoin binaries
-RUN set -ex \
-	&& cd /tmp \
-	&& wget -qO bitcoin.tar.gz "$BITCOIN_URL" \
-	&& tar -xzvf bitcoin.tar.gz -C /usr/local --strip-components=1 --exclude=*-qt \
-	&& rm -rf /tmp/*
+ARG SRC_IMAGE=zquestz/bitcoin-abc
+ARG SRC_IMAGE_TAG=latest
```

Comment:
Seems zquestz is this guy https://github.com/zquestz? Can we trust him? If not, this might pose a security vulnerability as he can upload any image to his repo. Even if we use specific tag instead of latest he can re-upload the image. If it's not an official image I'd rather revert it back.

Reply:
I checked zquestz only after I merged the PR :)

Reply:
> Seems zquestz is this guy https://github.com/zquestz? Can we trust him? 

The only thing I see is his images are referenced from [offsite](https://www.bitcoinabc.org). They look pretty regular. According to [bitcoinabc doc](https://github.com/Bitcoin-ABC/bitcoin-abc/blob/master/doc/release-process.md#release) :
```Notify maintainers of Ubuntu PPA, AUR, and Docker images to build their packages. They should be given 1-day advance notice if possible.``` 
they notify maintainers around release. I'm not sure how are maintainers connected to bitcoinabc org. I see zquestz Josh Ellithorpe is connected to bitcoin-cash, but unable to find any official statements. Here is [PR](https://github.com/Bitcoin-ABC/bitcoinabc.org/pull/67) where links to packages were added. 

We don't use ubuntu packages in our own docker image, we have direct `wget` download from offsite instead. But yes, there is no problem to push any malicious image with any tag into controlled dockerhub account.

Reply:
ok if it's reference from official site and github it's quite trustworthy.